components/contribution-flow/ContributionFlowSuccess.js

```diff
@@ -234,6 +234,11 @@ class ContributionFlowSuccess extends React.Component {
       );
     }
 
+    console.log('data', data);
+    console.log('collective', collective);
+    // TODO: For an event, most of the time parameters appear to be undefined -- not sure how to pass event time params to ContributorCard.
+    console.log('collective time', collective.startsAt, collective.endsAt, collective.timezone, collective.isSameDay);
```

Comment:
@Betree can you advise as to how I should be accessing the event's time parameters to pass to the calendar invite? 

I checked out `HeroEventDetails` and expected the `collective` variable here to similarly provide the event's start and end time (etc), but it looks like it does not (for several test events, the `startsAt`, `timezone`, and `isSameDay` variables were `undefined`).

thanks!

Reply:
`startsAt` is the right property. You must make sure all the fields you want to use are fetched from the success page, which relies on [this fragment](https://github.com/opencollective/opencollective-frontend/blob/3e10a2a42c2ec30e1b027b8889ebfe77c7183fb3/components/contribution-flow/graphql/fragments.js#L116).